### PR TITLE
Race optimization page

### PR DESCRIPTION
**Pull request by bill-luu**

### CODE REVIEW

File: web-app/src/app/app.module.ts [hunk 1]
```diff
@@ -52,7 +57,7 @@ import { APP_INITIALIZER } from '@angular/core';
     MatInputModule,
     MatProgressBarModule,
     MatProgressSpinnerModule,
-    MatTabsModule
+    MatTableModule,
```
Comment (nathanaelhuh) [inline web-app/src/app/app.module.ts]: Why did we change this again?

File: web-app/src/app/leftpanel/leftpanel.component.ts [hunk 2]
```diff
@@ -14,21 +14,23 @@ export class LeftpanelComponent implements OnInit {
...
   }
 
   ngOnInit() {
     // initialize with default values
     this.packet = this.packetService.getData();
-
+    this.timestamp = new Date(Number(this.packet.timestamp))
     // observe changes and update public variable when changed
     // note the dollar sign, this means you can subscribe to the object
     // see _services/packet.service.ts
```
Comment (nathanaelhuh) [inline web-app/src/app/leftpanel/leftpanel.component.ts]: the comments for my timestamp are also here

File: web-app/src/app/app.module.ts (hunk 1 above)
Comment (bill-luu) [reply web-app/src/app/app.module.ts]: Because of the angular update

File: web-app/src/app/leftpanel/leftpanel.component.ts (hunk 2 above)
Comment (bill-luu) [reply web-app/src/app/leftpanel/leftpanel.component.ts]: Did you want to delete them?

**Comment by bill-luu** [comment]
Leaving merge rights for Nathanael

**Pull request merged**